### PR TITLE
Share button in Step 3 for each collection

### PR DESCRIPTION
**Pull request by pmbohm**
Uses font-awesome as used by the Snazo branch, however this is the latest release of font-awesome.

### CODE REVIEW

**Comment by danfruehauf** [comment]
Did you branch this off snazo?

**Comment by pmbohm** [comment]
No Travis failed without any of my efforts.
"ERROR: An error occured while trying to fetch your .travis.yml file.
Is GitHub down? Please contact support@travis-ci.com if this persists."

**Comment by pmbohm** [comment]
err actually I have errors

**Comment by dnahodil** [comment]
This PR is asking to put the changes back to master (ie. not back onto snazo branch). I assume that's a mistake. (I think this was the point on @danfruehauf's comment)

**Comment by pmbohm** [comment]
Geez Monday morning. No this is meant for master @dnahodil @danfruehauf

**Comment by dnahodil** [comment]
There is definitely some miscommunication going on here. See below from @pblain. Let's sort this out in the review.

> Hi Guys,
>
> Please don't merge into master.  JPSR is still discussing the issue.
> 
> Cheers,
> Pete.

**Comment by pmbohm** [comment]
"Uses font-awesome as used by the Snazo branch" thats all.
Its "2 commits into master from shareCollectionLinks"
Its not Snazo. Its only "Share button in Step 3 for each collection"

The word Snazo is scaring you @dnahodil sorry

File: web-app/css/general.css [hunk 1]
```diff
@@ -1437,3 +1433,7 @@ div.allwhite {
     overflow: hidden;
     text-overflow: ellipsis;
 }
+
+
+
+
```
Comment (dnahodil) [inline web-app/css/general.css]: Extra spaces added here.

**Comment by dnahodil** [comment]
The word snazo was *confusing* me.

Just one comment, @pmbohm. Then good to merge.

**Comment by pmbohm** [comment]
@dnahodil Code tidy done

**Comment by danfruehauf** [comment]
Can be all squashed to one commit IMO.

**Comment by dnahodil** [comment]
Thanks @pmbohm. Will merge when travis is happy.

**Pull request merged**